src/redux/studio.js

```diff
@@ -72,6 +78,8 @@ const setRoles = roles => ({
     roles: roles
 });
 
+// Thunks
```

Comment:
What is a thunk?

Reply:
iirc, asynchronous action (see redux-thunk)